product_docs/docs/migration_portal/4/03_mp_using_portal/mp_ai_copilot/ai_good_prompts.mdx

```diff
@@ -3,21 +3,21 @@ title: How to create good prompts
 
 ---
 
-The quality and accuracy of the answers the AI Copilot provides depends highly on your prompt. 
+The quality and accuracy of the answers that the AI Copilot provides depend on the quality of your prompt. 
 The AI Copilot works best if it has large context around a question, details or specifics about the situation, and examples. 
-The more detailed and precise you can make your prompt, the better answers you will obtain. 
+The more detailed and precise you can make your prompt, the better the answers will be. 
 
 Like with other language models, you can use prompt engineering techniques with the AI Copilot to improve your query input. 
 
 !!! Note Data considerations
-    All prompts inserted in the chatbox are stored in an EDB-managed backend database and stored by the AI service provider.
+    All prompts inserted in the chatbot are stored in an EDB-managed backend database and stored by the AI service provider.
```

Comment:
This should either have been chat box or chatbot, so I opted for chatbot to be consistent with other usages. But if you meant box, please make it two words.